app/Mod.php

```diff
@@ -84,6 +84,11 @@ public function loadouts()
         return $this->belongsToMany(Loadout::class, 'loadout_mod');
     }
 
+    public function getImageSvgAttribute()
+    {
+        return "/assets/mods/optimized/{$this->icon}.svg";
```

Comment:
We are missing a few icons from this folder, I will push those and then we should be good

Reply:
Fixed in https://github.com/drg-tools/karl.gg/pull/323/commits/e85c3125da467b842c1ddab2324af98de8a31fa8
 
It is also fixed in stage so confirmed this fixed the issue (tested locally too)